iqmath/examples/get_started/pytest_iqmath_example.py

```diff
@@ -2,5 +2,6 @@
 
 
 @pytest.mark.generic
+@pytest.mark.parametrize('target', ['esp32'], indirect=True)
```

Comment:
this example doesn't depend on any target, so doesn't have to force it run only with esp32.


```suggestion
```